### PR TITLE
Adding pxc57 molecule components for package-testing

### PR DESCRIPTION
**Pull request by panchal-yash**
Add AMIs for pxc57 for ps80 jenkins server

Change instance type to t2.small

Add oracle linux 8 instead of centos 8

Adding python3-mysqldb for ubuntu-bionic, debian-bullseye

adding throttle: 1 for starting and running bats

### CODE REVIEW

File: molecule/pxc/pxc57-common/molecule/vagrant/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  playbooks:
+    converge: ../../playbooks/playbook.yml
+    prepare: ../../../playbooks/vagrant.yml
+  inventory:
+    host_vars:
+      pxc2-57-common:
+        ip_env: PXC2_IP
+      pxc3-57-common:
+        ip_env: PXC3_IP
+        sleep_before_tests: 40
```
Comment (eleo007) [inline molecule/pxc/pxc57-common/molecule/vagrant/molecule.yml:42]: Indentation - new line.

File: molecule/pxc/pxc57-common/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    when: ansible_os_family == "Debian"
+
+  - name: install python mysql deb packages
+    apt:
+      name: "{{ packages }}"
+      update_cache: yes
+    vars:
+      packages:
+      - python-mysqldb
+    when: (ansible_os_family == "Debian" and ansible_distribution_release != "focal") and (ansible_os_family == "Debian" and ansible_distribution_release != "bionic") and (ansible_os_family == "Debian" and ansible_distribution_release != "bullseye")
```
Comment (eleo007) [inline molecule/pxc/pxc57-common/tasks/main.yml:49]: The same as for bootstrap

**Review by eleo007 — CHANGES_REQUESTED** [review]
PXC 57 tests have 'pxc57_repo' option. PXC57 packages are stored in 2 repos: original and pxc57. So we need to move this functionality here - to select in from what repo to install packages for test.

File: molecule/pxc/pxc57-bootstrap/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    when: ansible_os_family == "Debian"
+
+  - name: install python mysql deb packages
+    apt:
+      name: "{{ packages }}"
+      update_cache: yes
+    vars:
+      packages:
+      - python-mysqldb
+    when: (ansible_os_family == "Debian" and ansible_distribution_release != "focal") and (ansible_os_family == "Debian" and ansible_distribution_release != "bionic") and (ansible_os_family == "Debian" and ansible_distribution_release != "bullseye")
```
[the remark this answers is not in the document]
Comment (panchal-yash) [reply molecule/pxc/pxc57-bootstrap/tasks/main.yml:53]: Yes I shortened them using the following condition and tested it.

when: (ansible_os_family == "Debian") and (ansible_distribution_release != "focal" and ansible_distribution_release != "bionic" and ansible_distribution_release != "bullseye")

**Review by eleo007 — APPROVED** [review]

**Pull request merged**